src/ralph/sim_cards/models.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from django.conf import settings
+
+from ralph.assets.models.base import BaseObject
+from ralph.lib.mixins.models import AdminAbsoluteUrlMixin
+from ralph.back_office.models import Warehouse
+from ralph.lib.mixins.models import NamedMixin
+
+
+class CellularCarrier(AdminAbsoluteUrlMixin, NamedMixin, models.Model):
+    pass
+
+class SIMCard(AdminAbsoluteUrlMixin, models.Model):
+    pin1 = models.CharField(max_length=8)
```

Comment:
why we don't allow for empty pin?

Reply:
+1 Good point. Pin is required when you buy a new simcards in Poland but isn't required in every country. 
Done.